docs/ADR-43-signed-fetch.md

```diff
@@ -0,0 +1,131 @@
+# Signed Fetch
+
+## Context and Problem Statement
+
+We want to integrate our dApps with the explorer in order to provide a better experience for our users. To do so we need to send request authenticated as a wallet.
+
+## Considered options
+
+### Alt 1
+
+Ask the user to sign each request
+
+#### Pros
+
+- Users will known each time an scene will interact in their name
+- No intermediate code is required
+
+#### Cons
+
+- For privates APIs it will require to many interactions for the users, potentially degrading the experience within the explorer
+
+### Alt 2
+
+Use [`decentraland-crypto`](https://github.com/decentraland/decentraland-crypto) as intermediate to sign each request
+
+#### Pros
+
+- It only requires one sign from the user
+- It is already integrated in the explorer
+  - [`signedFetch`](https://docs.decentraland.org/development-guide/network-connections/#signed-requests)
+- Libraries to interact with these firms are already published
+  - [`decentraland-crypto`](https://github.com/decentraland/decentraland-crypto)
+  - [`decentraland-crypto-middleware`](https://github.com/decentraland/decentraland-crypto-middleware)
+  - [`decentraland-crypto-fetch`](https://github.com/decentraland/decentraland-crypto-fetch)
+
+#### Cons
+
+- It requires an extensive documentation
+- It requires extra code
+
+## Decision
+
+The option to use [`decentraland-crypto`](https://github.com/decentraland/decentraland-crypto) to interact with our services was chosen in order to provide a better experience to our users
+
+## How to sign a request
+
+### Generate a payload
+
+In order to minimize the reutilization of request each payload must include data about the request and the moment it is made
+
+```typescript
+
+/**
+ * Request method
+ */
+const METHOD: "GET" | "POST" | "PUT" | "PATCH" | "DELETE" = `POST`
+
+/**
+ * request path without domain, query string or hash
+ * @example
+ * - `/ping`
+ * - `new URL('https://decentraland.org/ping').pathname`
+ */
+const PATH: string = `/ping`
+
+/**
+ * request timestamp
+ * each service decide how long will consider valid a request using this timestamp,
+ * if timestamp is greater than now the request should fail
```

Comment:
should the services decide on when the signature is too old, or should we use an expiration date signed by the user? to me the later sounds more transparent.